app/src/main/kotlin/com/wire/android/model/PreservedState.kt

```diff
@@ -0,0 +1,15 @@
+package com.wire.android.model
+
+import com.wire.kalium.logic.CoreFailure
+
+sealed class PreservedState<out T : Any> {
```

Comment:
Same here, could be a good idea to add some docs here, with the intended purpose of this class :D

Reply:
Sure 👍